policyengine_us/parameters/gov/states/az/tax/income/deductions/standard/rate.yaml

```
@@ -0,0 +1,17 @@
+description: Arizona increases the standard deduction amount by this rate of charitable contributions.
+metadata:
+  label: Arizona increased standard deduction rate
+  period: year
+  unit: currency-USD
+  reference:
+  - title: 2022 Arizona Form 140 Resident Personal Income Tax Booklet
+    href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140BOOKLET.pdf#page=20
```

Needs to be page of pdf rather than the page number, check other too
```suggestion
    href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140BOOKLET.pdf#page=24
```